source/features/mark-merge-commits-in-list.tsx

```diff
@@ -9,6 +9,8 @@ import batchedFunction from 'batched-function';
 import observe from '../helpers/selector-observer.js';
 import features from '../feature-manager.js';
 import api from '../github-helpers/api.js';
+import {commitHashLinkInLists, commitTitleInLists} from '../github-helpers/selectors.js';
+import {assertCommitHash} from '../github-helpers/index.js';
```

Comment:
The best way to keep track of selectors, especially grouped ones being applied to multiple pages, is by testing them.